projects/akita-ng-fire/src/lib/factory/factory.service.ts

```diff
@@ -125,7 +184,7 @@ export class FactoryService<S extends EntityState<any, string>> {
   ): Promise<void>;
   async update(
     idOrEntity: string | Partial<getEntityType<S>>,
-    newStateOrOption?: Partial<getEntityType<S>> | WriteOptions,
+    newStateOrOption?: Partial<getEntityType<S>>,
```

Comment:
I don't understand why you remove `WriteOptions` here. You should be able to do :
```
service.update({ id: '1', title: 'new title' }, { write: batch });
```

Reply:
6949b37